pkg/setting/provider.go

```diff
@@ -115,6 +119,20 @@ func (o OSSImpl) Current() SettingsBag {
 	return settingsCopy
 }
 
+func (o OSSImpl) CurrentVerbose() VerboseSettingsBag {
+	settingsCopy := make(VerboseSettingsBag)
+
+	for _, section := range o.Cfg.Raw.Sections() {
+		settingsCopy[section.Name()] = make(map[string]map[string]string)
+		for _, key := range section.Keys() {
+			settingsCopy[section.Name()][key.Name()] = make(map[string]string)
+			settingsCopy[section.Name()][key.Name()]["value"] = RedactedValue(EnvKey(section.Name(), key.Name()), key.Value())
```

Comment:
shouldn't the "file" key also be added here.

Also could you export the constants "value" and "file" here to be used by overriding services? 

(make sure the services don't become misaligned)

Reply:
Changed as requested. I'd appreciate feedback on it!